cadc-cdp-server/build.gradle

```diff
@@ -25,8 +25,9 @@ dependencies {
     compile 'org.bouncycastle:bcprov-jdk15on:1.46'
     compile 'javax.servlet:javax.servlet-api:3.1.0'
 
-    compile 'org.opencadc:cadc-util:[1.8.4,2.0)'
-    compile 'org.opencadc:cadc-cdp:[1.3,2.0)'
+    compile 'org.opencadc:cadc-util:[1.8.4,1.11)'
```

Comment:
Is there an advantage to not pin it to `cadc-util 1.10`? And the comment applies to both of them.

Reply:
fixed